worldedit-core/src/main/java/com/fastasyncworldedit/core/Fawe.java

```diff
@@ -86,6 +89,15 @@ public class Fawe {
      * The platform specific implementation.
      */
     private final IFawe implementation;
+    private final KeyQueuedExecutorService<UUID> clipboardExecutor = Settings.settings().CLIPBOARD.USE_DISK
```

Comment:
Maybe initialize this in the ctor for consistency? But also same discussion as in other places, other plugins might use clipboard on disk features without the setting being enabled.